src/esp/sim/Simulator.h

```diff
@@ -608,6 +608,25 @@ class Simulator {
   void setObjectLightSetup(int objectID,
                            const std::string& lightSetupKey,
                            int sceneID = 0);
+  
+  int findNearestObject(int refObjectID, float distance=1.0);
```

Comment:
Please consider updating the Python bindings for these new functions too

Reply:
I am using the code that @dexter1691 created for actions grab/release. He is planning to submit these changes with python bindings in a separate PR.

Reply:
Fair, but we should finalize the bindings so that they agree. Either please submit a micro-PR with those bindings ASAP @dexter1691, or just add the bindings for him. If they are the same, it shouldn't affect his PR.